DynamicLights/src/main/java/atomicstryker/dynamiclights/server/datagen/ModBlockStateProvider.java

```diff
@@ -0,0 +1,21 @@
+package atomicstryker.dynamiclights.server.datagen;
+
+import atomicstryker.dynamiclights.server.DynamicLights;
+import net.minecraft.data.PackOutput;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.model.generators.BlockStateProvider;
+import net.minecraftforge.common.data.ExistingFileHelper;
+
+public class ModBlockStateProvider extends BlockStateProvider {
```

Comment:
comment would be nice as to what the purpose of this class is

Reply:
Comment added.